python/packages/autogen-ext/src/autogen_ext/models/_openai/_openai_client.py

```diff
@@ -86,10 +86,29 @@ def _azure_openai_client_from_config(config: Mapping[str, Any]) -> AsyncAzureOpe
     # Take a copy
     copied_config = dict(config).copy()
 
+    import warnings
```

Comment:
In `AzureOpenAIChatCompletionClient` constructor `model` argument is still checked:

```python
        if "model" not in kwargs:
            raise ValueError("model is required for OpenAIChatCompletionClient")
```

Reply:
We also need to update the API documentation by changing the example in the doc string.

Reply:
We need to update usage of AzureOpenAIChatCompletionClient in the documentation to change `model` to `azure_deployment`.

Reply:
Model is required in the create call so `model` and `azure_deployment` are both needed in the call. However, this means we can most likely go ahead and remove the requirement to always specify model capabilities